pkg/api/v1beta3/dynakube/feature_flags.go

```diff
@@ -129,9 +129,9 @@ func (dk *DynaKube) FeatureNoProxy() string {
 	return dk.getFeatureFlagRaw(AnnotationFeatureNoProxy)
 }
 
-// FeatureActiveGateAutomaticTLSCertificate is a feature flag to enable automatic creation of AG TLS certificate if TrustedCAs are used.
+// FeatureActiveGateAutomaticTLSCertificate is a feature flag to disable automatic creation of ActiveGate TLS certificate.
```

Comment:
Out of curiosity: Why is this needed here? Wouldn't be a conversion the better approach here?

Reply:
The FF is present in v1beta3 on the main branch only. It was a "temporary" home for it until v1beta4 was introduced. 
I would like them to be consistent with each other to avoid potential confusion.